spread.yaml

```diff
@@ -43,7 +43,7 @@ environment:
     TRUST_TEST_KEYS: '$(HOST: echo "${SPREAD_TRUST_TEST_KEYS:-true}")'
     # a global setting for LXD channel to use in the tests
     # TODO: Consider reverting to latest/candidate after Snapcraft compatibility with LXD 5.21 extended version string "5.21 LTS" is fixed
-    LXD_SNAP_CHANNEL: "latest/edge"
+    LXD_SNAP_CHANNEL: "latest/candidate"
```

Comment:
There are more things to revert, see: https://github.com/snapcore/snapd/pull/13724/files

(1) LXD | spread.yaml  - change back to candidate - DONE
(2) LXD| tests/nested/manual/devmode-snaps-can-run-other-snaps/task.yaml - remove "LXD_SNAP_CHANNEL: 5.20/stable"
(3) SNAPCRAFT | tests/lib/tools/setup_nested_hybrid_system.sh - change back to "snap install snapcraft --candidate --classic"
(4) SNAPCRAFT |  tests/nested/manual/muinstaller-core/task.yaml - change back to "snap install snapcraft --candidate --classic"

Reply:
done

Reply:
Strictly speaking this line is not being reverted as we are using candidate instead of stable as before the change. But that is fine.